arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -1,240 +1,157 @@
 {
   "swagger": "2.0",
   "info": {
+    "version": "2015-05-01-preview",
```

Comment:
I've added attributes for Advisors and Recommended Actions controllers. could you generate this file again?